README.md

```diff
@@ -1,39 +1,61 @@
-RaceCapture/Pro Firmware
-========================
+RaceCapturePro Firmware
+=====
 
-Firmware for RaceCapture/Pro data acqusition / control / telemetry system
+The premier open source telemetry system for your race vehicle.  RaceCapturePro is a
+device that gets installed into your race vehicle and beams telemetry about your
+vehicle and driving habits in real time.  Combined with
+[RaceCapture App](https://github.com/autosportlabs/RaceCapture_App) and
```

Comment:
Suggest: 

A powerful and flexible open source telemetry system for motorsport applications. RaceCapture/Pro installs into your race vehicle and transmits real-time wireless telemetry data via GSM cellular and Bluetooth.  Combined with